.storybook/components/SidebarLabelWrapper/SidebarLabelWrapper.js

```diff
@@ -17,6 +17,7 @@ const cmpStages = {
 };
 
 const { versions } = libVersions;
+const [lastVersion] = [...versions].reverse();
```

Comment:
You can use here 
```suggestion
const [lastVersion] = versions.toReversed();
```
To avoid mutation without spreading

or  `const lastVersion = versions.at(-1);`

Reply:
The suggested `toReversed` method is a good idea. An approach with a negative index is not the best practice from the point of view of code readability. But under the hood, the `toReversed` method also allocates extra memory for reverse without mutation so what is the benefit to changing the code? as this statement executes once when the module is loaded.